borg/archiver.py

```diff
@@ -1533,6 +1533,11 @@ def build_parser(self, args=None, prog=None):
 
         To allow a regular user to use fstab entries, add the ``user`` option:
         ``/path/to/repo /mnt/point fuse.borgfs defaults,noauto,user 0 0``
+
+        The BORG_MOUNT_DCACHE_ENTRIES environment variable is meant for advanced users
```

Comment:
Ha, I thought about icache/dcaches of cpus, not the Kernel, but yeah, since this is an FS it's confusing.